code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -22,6 +22,7 @@
 	var/elimination = 0
 	anchored = TRUE
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
+	mouse_opacity = 2 // Easier to click on in melee, they're giant targets anyway
```

Comment:
Legion's sprite is super huge and I'm not sure how mouse_opacity interacts with transform scaling(hopefully sanely but I've never tested it and you never know with byond)
Also in general I don't think this is a good idea because A: it means you can't click on stuff near the megafauna B: I don't think they actually need to be easier to click? They're huge mobs already. C: player ash drakes will probably need mouse_opacity 1 so they're actually playable instead of frustrating